### PR TITLE
Prepare for 0.11.0 release

### PR DESCRIPTION
**Pull request by urschrei**
- [x] I agree to follow the project's [code of conduct](https://github.com/georust/geo/blob/master/CODE_OF_CONDUCT.md).
- [ ] I added an entry to `rstar/CHANGELOG.md` if knowledge of this change could be valuable to users.
---

### CODE REVIEW

**Review by adamreichold — APPROVED** [review]

File: rstar/Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "rstar"
-version = "0.10.0"
+version = "0.11.0"
```
Comment (michaelkirk) [inline rstar/Cargo.toml:3]:
```suggestion
version = "0.10.1"
```

File: rstar/CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# UNRELEASED
+# 0.11.0
```
Comment (michaelkirk) [inline rstar/CHANGELOG.md:1]:
```suggestion
# 0.10.1
```

**Review by michaelkirk — COMMENTED** [review]
I *think* these are all non-breaking changes, right?

**Comment by adamreichold** [comment]
> I _think_ these are all non-breaking changes, right?

Technically, yes. But we did bump our MSRV. Personally, I find it impolite to do this in a patch release.

**Comment by urschrei** [comment]
@michaelkirk Yes my bad, but I think Adam makes a good point about the MSRV

**Comment by rmanoka** [comment]
Changes lgtm;  agree with bumping to `0.11.0` as some compilation might break elsewhere.

**Review by rmanoka — APPROVED** [review]

**Review by michaelkirk — APPROVED** [review]
The "should MSRV be breaking" debate is pretty well trodden territory. I'm in the "it does more harm than good" camp, but I won't die on that hill this time.

I will note that this change brings us past the introduction of `rust-version` (introduced in 1.56), so future bumps to `rust-version` have no case for being a breaking semver bump as I understand it.

**Review by michaelkirk — APPROVED** [review]
The "should MSRV be breaking" debate is pretty well trodden territory. I'm in the "it does more harm than good" camp, but I won't die on that hill this time.

I will note that this change brings us past the introduction of `rust-version` (introduced in 1.56), so future bumps to `rust-version` have no case for being a breaking semver bump as I understand it.

**Comment by urschrei** [comment]
> The "should MSRV be breaking" debate is pretty well trodden territory. I'm in the "it does more harm than good" camp, but I won't die on that hill this time.
> 
> I will note that this change brings us past the introduction of `rust-version` (introduced in 1.56), so future bumps to `rust-version` have no case for being a breaking semver bump as I understand it.

I for one am delighted that we can settle this issue using metadata.

**Pull request merged**